worlds/spraybot.world

```diff
@@ -9,7 +9,231 @@
+      <name>'tree_6_0'</name>
+      <pose>-3.04800 3.57000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_6_1'</name>
+      <pose>-1.52400 3.57000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_6_2'</name>
+      <pose>0.00000 3.57000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_6_3'</name>
+      <pose>1.52400 3.57000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_6_4'</name>
+      <pose>3.04800 3.57000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_7_0'</name>
+      <pose>-3.04800 5.10000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_7_1'</name>
+      <pose>-1.52400 5.10000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_7_2'</name>
+      <pose>0.00000 5.10000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_7_3'</name>
+      <pose>1.52400 5.10000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_7_4'</name>
+      <pose>3.04800 5.10000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_8_0'</name>
+      <pose>-3.04800 6.62000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_8_1'</name>
+      <pose>-1.52400 6.62000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_8_2'</name>
+      <pose>0.00000 6.62000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_8_3'</name>
+      <pose>1.52400 6.62000 0.00000 0 0</pose>
+    </include>
+    <include>
+      <uri>https://fuel.ignitionrobotics.org/1.0/shrijitsingh99/models/Juniper Tree</uri>
+      <name>'tree_8_4'</name>
+      <pose>3.04800 6.62000 0.00000 0 0</pose>
+    </include>
     <spherical_coordinates>
```

Comment:
Could we move `spherical_coordinates` to the top of the file, so its more visible. If the number of trees grows, it will get buried path the bottom and forgotten.